leo/core/leoserver.py

```diff
@@ -2110,11 +2110,11 @@ def goto_global_line(self, param: Param) -> Response:
         gc = c.gotoCommands
         line = param.get('line', 1)
         try:
-            junk_p, junk_offset, found = gc.find_file_line(n=int(line))
+            p, junk_offset = gc.find_file_line(n=int(line))
+            focus = self._get_focus()
```

Comment:
Great fix for the server too, thanks! But _get_focus should remain outside the try-except block as it's method body is a try-except block itself already.

Reply:
I agree. `_get_focus` gives a more specific error.